deploy_chain.sh

```diff
@@ -15,7 +15,7 @@ chains=(
     ["AVALANCHE"]=31339
     ["POLYGON"]=31340
     ["CRONOS"]=31341
-    ["FANTOM"]=31342
+    ["SONIC"]=31342
```

Comment:
Let's put a comment explaining the name change and the date this change was done.